pkg/services/accesscontrol/resourcepermissions/service.go

```diff
@@ -77,9 +79,36 @@ func New(cfg *setting.Cfg,
 		actions = append(actions, action)
 	}
 
+	// create the store for actionsets
+	log := log.New("accesscontrol.resourcepermissions")
+	actionSetsStore := NewInMemoryActionSets(log)
```

Comment:
nit: I think we'll probably want to pass action set store as a dependency to this method instead of creating it here. But we can do that later.

Reply:
leaving a comment for that